src/components/DashboardLayout/SettingsLeftNav.tsx

```diff
@@ -1,51 +1,110 @@
-import { useContext } from 'react';
+import { useContext, useState, useEffect } from 'react';
 import Button from 'components/Atoms/Button';
-import { faXmark } from '@fortawesome/free-solid-svg-icons';
+import { faXmark, faServer, faRightToBracket } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { BrowserLocationContext } from 'data/BrowserLocationContext';
 import { RadioGroup } from '@headlessui/react';
 import { SettingsContext } from 'data/SettingsContext';
 import Avatar from 'components/Atoms/Avatar';
 import clsx from 'clsx';
 import { useUid } from 'data/UserInfo';
+import CoreSettings from 'components/Settings/CoreSettings';
+import UserSettings from 'components/Settings/UserSettings';
+
+
+export const tabList = [
+    {
+    title: 'General',
+      path: 'general',
+      content: <CoreSettings />,
+    },
+    {
+      title: 'Users',
+      path: 'users',
+      content: <UserSettings />,
+    },
+];
 
 export default function SettingsLeftNav({ className }: { className?: string }) {
+  const [setActive, setActiveTab] = useState(location.pathname.split('/')[2]);
   const { setPathname, setSearchParam } = useContext(BrowserLocationContext);
-  const { userList, selectedUser, selectUser, tabIndex } =
-    useContext(SettingsContext);
+  const { userList, selectedUser, selectUser} =
+  useContext(SettingsContext);
   const uid = useUid();
+
+  useEffect(() => {
+    setActiveTab(location.pathname.split('/')[2]);
```

Comment:
probably better to use a `useMemo`
it would be nice if you can refactor the same thing in `SelectedInstanceInfo`